src/new-components/heading/heading.module.css

```diff
@@ -46,15 +53,48 @@ h5.heading,
 h6.heading {
     /*
      * unlike at higher levels, this one is kept as the same size as h3's
-     * you can make it one level larger visually, but making it smaller has no effect
+     * you can make it two levels larger visually, but making it smaller has no effect
      */
     font-size: var(--reactist-font-size-normal);
 }
 
+h4.size-largest,
+h5.size-largest,
+h6.size-largest {
+    font-size: var(--reactist-font-size-header);
+}
+
 h4.size-larger,
 h5.size-larger,
 h6.size-larger {
     font-size: var(--reactist-font-size-large);
 }
 
 /* h4/h5/h6 can't be made smaller, maybe we reconsider this? */
+
+/* truncated text */
```

Comment:
Yeah .line-clamp-1 applies `text-overflow: ellipsis;` instead of `-webkit-line-clamp`.

The vendor prefixes are required since autoprefixer doesn't handle this for now: https://github.com/postcss/autoprefixer/issues/1322